defaults/main.yml

```diff
@@ -27,3 +27,5 @@ nginx_sites:
 nginx_configs: {}
 
 nginx_auth_basic_files: {}
+
+install_rpm_repositories: "true"
```

Comment:
It's only for epel repository, so I think a better var name could be `install_rpm_epel_repository`